apps/backoffice-v2/src/lib/blocks/components/AmlBlock/hooks/useAmlBlock/useAmlBlock.tsx

```diff
@@ -89,7 +89,11 @@ export const useAmlBlock = (data: Array<TWorkflowById['context']['aml']>) => {
             type: 'dataTable',
             value: {
               data: matches,
-              props: { scroll: { className: 'h-[50vh]' }, cell: { className: '!p-0' } },
+              props: {
+                scroll: { className: totalMatches === 0 ? 'h-[10vh]' : 'h-[50vh]' },
```

Comment:
Not a must but remember `ctw`'s main use-case is conditional classes. Something like `ctw('h-[50vh]', { 'h-[10vh]: totalMatches === 0 }')`